### PR TITLE
Updating oc-specific descriptions

### PR DESCRIPTION
**Pull request by bergerhoffer**
Minor grammatical and punctuation consistency updates to oc-specific CLI commands. Short descriptions and comments in examples do not use ending punctuation; long descriptions do use ending punctuation.

https://issues.redhat.com/browse/OSDOCS-2018
https://issues.redhat.com/browse/OSDOCS-1790

### CODE REVIEW

File: pkg/cli/extract/extract.go [hunk 1]
```diff
@@ -23,30 +23,30 @@ import (
...
-		Extract files out of secrets and config maps
+		Extract files out of secrets and config maps.
 
 		The extract command makes it easy to download the contents of a config map or secret into a directory.
 		Each key in the config map or secret is created as a separate file with the name of the key, as it
 		is when you mount a secret or config map into a container.
 
-		You may extract the contents of a secret or config map to standard out by passing '-' to --to. The
-		names of each key will be written to stdandard error.
+		You may extract the contents of a secret or config map to standard out by passing '-' to --. The
```
Comment (bergerhoffer) [inline pkg/cli/extract/extract.go:32]: @soltysh Can you make sure that the removal of the second "to" is correct here?

Comment (soltysh) [reply pkg/cli/extract/extract.go:32]: No, `--to` is a flag so it has to stay as is.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**